runtime/queries/lua/highlights.scm

```diff
@@ -70,36 +70,20 @@
   ] @keyword.function)
 
 ; Operators
+(binary_expression
+  operator: _ @operator)
+
+(unary_expression
+  operator: _ @operator)
+
+"=" @operator
+
 [
   "and"
   "not"
   "or"
 ] @keyword.operator
 
-[
-  "+"
-  "-"
```

Comment:
Removing this pattern is what triggers the failures, but I don't see how that enters into the failing tests.